lib/cogctl.ex

```diff
@@ -33,17 +33,29 @@ defmodule Cogctl do
   end
 
   defp configure_identity(options, config) do
-    profile =
-      case try_profiles(options, config) do
-        nil ->
-          default_profile
-        profile ->
-          profile
-      end
+    profile = case try_profiles(options, config) do
+                nil -> default_profile
+                profile -> profile
+              end
+    profile = apply_overrides(profile, options)
+    client = new_client(profile)
```

Comment:
This is where we call `new_client` now, since it was being called every time anyway.